wp-content/themes/wp-starter/src/styles/base/global.css

```diff
@@ -3,29 +3,37 @@
 		@apply !hidden;
 	}
 
-	ul {
-		@apply list-disc;
+	ul.wp-block-list {
+		@apply list-disc pl-32;
 
 		li {
-			@apply list-inside;
+			@apply list-outside;
 		}
 	}
 
-	ol {
-		@apply list-decimal;
+	ol.wp-block-list {
+		@apply list-decimal pl-32;
 
 		li {
-			@apply list-inside;
+			@apply list-outside;
 		}
 	}
+
+	a {
+		@apply underline;
+	}
+
+	p + p {
+		@apply mt-24;
+	}
```

Comment:
Do you prefer this vs `p:not(:empty):not(:last-child) { @apply mb-24; }`? This won't apply when images, lists, button groups, etc. surround paragraphs.

Reply:
Nope, lets go with `p:not(:empty):not(:last-child)`! 
`p + p` is the old school way of doing it, old habits die hard 😆